mintpy/closure_phase_bias.py

```diff
@@ -8,39 +8,106 @@
+    closure_phase_bias.py -i inputs/ifgramStack.h5 --nl 20 --bw 10 -a quick_biasEstimate
+    closure_phase_bias.py -i inputs/ifgramStack.h5 --nl 20 --bw 10 -a biasEstimate
 """
 
 def create_parser():
-    parser = argparse.ArgumentParser(description = 'Create an indication map for closure phase bias.')
+    parser = argparse.ArgumentParser(description = 'This script deals with phase non-closure related bias, either in terms of masking, estimation or correction.',
+                                    formatter_class = argparse.RawTextHelpFormatter,
+                                    epilog=REFERENCE+'\n'+EXAMPLE)
     parser.add_argument('-i','--ifgramstack',type = str, dest = 'ifgram_stack',help = 'interferogram stack file that contains the unwrapped phases')
     parser.add_argument('--nl', dest = 'nl', type = int, default = 20, help = 'connection level that we are correcting to (or consider as no bias)')
+    parser.add_argument('--bw', dest = 'bw', type = int, default = 10, help = 'bandwidth of time-series analysis that you want to correct')
     parser.add_argument('--numsigma',dest = 'numsigma', type = float, default = 3, help = 'Threashold for phase (number of sigmas,0-infty), default to be 3 sigma of a Gaussian distribution (assumed distribution for the cumulative closure phase) with sigma = pi/sqrt(3*num_cp)')
     parser.add_argument('--epi',dest = 'episilon', type = float, default = 0.3, help = 'Threashold for amplitude (0-1), default 0.3')
     parser.add_argument('--maxMemory', dest = 'max_memory', type = float, default = 8, help = 'max memory to use in GB')
+    parser.add_argument('--noupdate_CP',dest = 'update_CP', action = 'store_false', help = 'Use when no need to compute closure phases')
     parser.add_argument('-o', dest = 'outdir', type = str, default = '.', help = 'output file directory')
+    parser.add_argument('-a','--action', dest='action', type=str, default='create_mask',
+                        choices={'create_mask', 'quick_biasEstimate', 'biasEstimate'},
+                        help='action to take (default: %(default)s):\n'+
+                             'create_mask -  create a mask of areas susceptible to closure phase errors\n'+
+                             'quick_biasEstimate - estimate how bias decays with time, will output sequential closure phase files, and gives a quick and appximate bias estimation\n'
+                             'biasEstimate - estimate how bias decays with time, processed for each pixel on a pixel by pixel basis')
+    parser = arg_group.add_parallel_argument(parser)
     return parser
 
 def cmd_line_parse(iargs=None):
     parser = create_parser()
     inps = parser.parse_args(args = iargs)
     return inps
 
-# Obtain sum of consecutive complex sequential closure phase of connection n
-def cum_seq_closurePhase(SLC_list, date12_list_all, ifgram_stack, ref_phase, n, box):
+def seq_closurePhase(SLC_list, date12_list_all, ifgram_stack, ref_phase, n, box):
```

Comment:
Could you apply the following suggestions to all functions?
1. use the `snake_case`, such as `closure_phase`.
2. describe what the function does
3. describe all the input and output parameters

`asc_desc2horz_vert.py` is a good reference for the doc string format used in mintpy.

Reply:
Updated.